README.md

```diff
@@ -157,6 +162,68 @@ This URL will stay the same when we upload an updated version of the website. Be
 
 ## Usability Features
 
+### Uploading Files, Folders, Websites, and Arbitrary Data from stdin
+
+#### Files
+
+Use `swarm-cli` to upload a single file:
+
+```
+swarm-cli upload README.md
+```
+
+The command above will print a `/bzz` URL that may be opened in the browser, and if the file format is suitable, the browser will display it.
```

Comment:
```suggestion
The command above will print a `/bzz` URL that may be opened in the browser. If the browser is able to handle the file format then the file is displayed, otherwise it will be offered to be downloaded.
```